test/parallel/test-tls-check-server-identity.js

```diff
@@ -253,9 +253,9 @@ var tests = [
 ];
 
 tests.forEach(function(test, i) {
-  var err = tls.checkServerIdentity(test.host, test.cert);
-  assert.equal(err && err.reason,
-               test.error,
-               'Test#' + i + ' failed: ' + util.inspect(test) + '\n' +
-               test.error + ' != ' + (err && err.reason));
+  const err = tls.checkServerIdentity(test.host, test.cert);
+  assert.strictEqual(err && err.reason,
+                     test.error,
+                     `Test# ${i} failed: ${util.inspect(test)} \n
+                     ${test.error} != ${(err && err.reason)}`);
```

Comment:
Please avoid multi-line template literals, especially given that the additional whitespace will be included in the string.

```js
                     `Test# ${i} failed: ${util.inspect(test)} \n` +
                     `${test.error} != ${(err && err.reason)}`);
```